lib/cc/analyzer/issue_location_format_validation.rb

```diff
@@ -30,13 +30,13 @@ def valid_positions?(positions)
       def valid_position?(position)
         position &&
           (
-            (position["line"] && position["column"]) ||
-            position["offset"]
+            [position["line"], position["column"]].all? { |value| value.is_a?(Integer) } ||
+            position["offset"] && position["offset"].is_a?(Integer)
```

Comment:
I think it's worth introducing and seeing what shakes out. We might want to fix the known Radon bug first as it's an engine we auto-enable.